go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/SAP/go-hdb v0.14.1
 	github.com/Sectorbob/mlab-ns2 v0.0.0-20171030222938-d3aa0c295a8a
 	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
+	github.com/aerospike/aerospike-client-go v3.1.0+incompatible
```

Comment:
I'm a bit curious as to why this particular pseudo-version was chosen since v3.1.1 is available.

Reply:
Interesting, the `+incompatible` label is actually how go/mod sees the release: https://pkg.go.dev/github.com/aerospike/aerospike-client-go@v3.1.0+incompatible?tab=versions

Since `v3.1.1+incompatible` has been released, we should consider bumping it to that version (unless there's a particular reason not to)